### ISSUE TITLE
training and validation loss value is greater than 1 

### ISSUE THREAD

**Issue by jeromeyoon** [fchollet/keras#1495]
![loss](https://cloud.githubusercontent.com/assets/8522478/12415690/6e7624ac-bee0-11e5-8737-816b2d95b6cc.PNG)

Hi 

I'm training using Graph model. But training and validation loss value is greater than 0 (training loss value from model.fit and validation loss value from model.evaluate)
Is it normal?

**Comment by keunwoochoi** [comment]
It depends on your loss function and labels, but seems like normal by itself.
